code/go/internal/spec_test.go

```diff
@@ -14,6 +14,12 @@ import (
 
 func TestBundledSpecsForIntegration(t *testing.T) {
 	fs := spec.FS()
-	_, err := fs.Open("1/integration/spec.yml")
+	_, err := fs.Open("integration/spec.yml")
```

Comment:
Thanks, I will make the follow up.

Reply:
I have updated the pull request template in this repository, and opened PRs in the integrations and docs repositories. Also added a placeholder to avoid broken links: https://github.com/elastic/package-spec/blob/26925a80d08a39cd0d188ba81bc71b09464b1d04/versions/1/README.md